### DISCUSSION TITLE
Dark Mode Toggle Issue

### DISCUSSION THREAD

**Discussion by aydinza** [livewire/flux#480] [Feature Requests]
I'm not sure if it's more related to Flux or Livewire itself, but the toggler behaves unexpected.

First switched to dark mode, but some of the items suddenly turn back to light mode, then when I try the toggler again back to light mode, it does not update the data. I don't have any other ui element. Only flux. 

I believe this might be caused by the Jetstream dark mode. Anyone had the same issue?

**Comment by lonebadger99** [comment]
Can you show us the code for the toggler?
